snappy_pipeline/workflows/somatic_gene_fusion_calling/__init__.py

```diff
@@ -78,6 +79,30 @@
       path_ctat_resource_lib: REQUIRED
     defuse:
       path_dataset_directory: REQUIRED
+    arriba:
+      path_index: REQUIRED       # STAR path index (preferably 2.7.10 or later)
+      features: REQUIRED         # Gene features (for ex. ENCODE or ENSEMBL) in gtf format
```

Comment:
You need to include the `REQUIRED` tag in the comments before it goes to the user/local config.